app/api_1_0/articles.py

```diff
@@ -27,11 +28,17 @@ def articles():
     page = request.args.get('page', 1, int)
     page_size = request.args.get('page_size', 20, int)
     query = filter_params(request)
-
     db = RevisionDB(config={'host': config['default'].MONGO_HOST, 'port': config['default'].MONGO_PORT, 'username': config['default'].MONGO_USERNAME, 'password': config['default'].MONGO_PASSWORD})
-
+    handler = QueryHandler(db=db)
+    query = handler.get_articles_query(query)
+    revisions=db.articles(query, page, page_size)
+    articles=[]
+    for rev in revisions:
```

Comment:
@korenerok can you change the date format on the query instead of doing it manually?